src/coreclr/nativeaot/docs/compiling.md

```diff
@@ -1,10 +1,13 @@
 # Compiling with Native AOT
 
-This document explains how to compile and publish your project using Native AOT toolchain. First, please _ensure that [pre-requisites](prerequisites.md) are installed_. If you are starting a new project, you may find the [HelloWorld sample](https://github.com/dotnet/samples/tree/main/core/nativeaot/HelloWorld/README.md) directions useful.
+Please consult [documentation](https://docs.microsoft.com/dotnet/core/deploying/native-aot) for instructions how to compile and publish application.
 
-## Add ILCompiler package reference
+The rest of this document covers advanced topics only.
 
-To use Native AOT with your project, you need to add a reference to the ILCompiler NuGet package containing the Native AOT compiler and runtime. Make sure the `nuget.config` file for your project contains the following package sources under the `<packageSources>` element:
+
+## Using daily builds
```

Comment:
This section is essentially duplicating https://github.com/dotnet/installer#installers-and-binaries, which covers all kinds of `PublishXxx` scenarios (including `PublishAot`).

Reply:
I think this small duplication is ok to make the steps easier to follow. 

It is duplicated in https://github.com/dotnet/runtime/blob/main/docs/project/dogfooding.md#obtaining-nightly-builds-of-nuget-packages as well.